Server/Type/WebSocketServer.php

```diff
@@ -90,7 +91,7 @@ public function __construct(
         ServerPushHandlerRegistry $serverPushHandlerRegistry,
         LoggerInterface $logger = null
     ) {
-        $this->loop = $loop;
+        $this->loop = $loop->create();
```

Comment:
Don't create many event loop on each services, they are insulated so that break some feature who share the same event loop across services, should be reverted